src/adapters/marketplace/_resolver.js

```diff
@@ -0,0 +1,284 @@
+import V00_MarkeplaceAdapter from './v00'
+import V01_MarkeplaceAdapter from './v01'
+import {
+  parseListingId,
+  parseOfferId,
+  generateListingId,
+  generateOfferId,
+  generateNotificationId
+} from '../../utils/id'
+import {
+  Notification,
+  readStatus,
+  unreadStatus,
+  storeKeys
+} from '../../models/notification'
+
+class MarketplaceResolver {
+  constructor({ contractService, ipfsService, store }) {
+    this.adapters = {
+      '000': new V00_MarkeplaceAdapter({ contractService, ipfsService, store }),
```

Comment:
Did we say that moving forward we would only use V01 marketplace ? And that V00 was only for testing purposes ?

Reply:
Yeah. That just means that we will create all new listings with the V01 marketplace (which we are already doing.) If we don't include that adapter here in the resolver, it might as well not be in the codebase.

That said, I was thinking some more and it may just be simplest to not do our first release with unnecessary versions. Depending on how others feel, I may do a separate PR to remove the extra version.

Reply:
+1 on removing V00. I don't see it adding anything, and it seems to be a source of ongoing confusion.

Reply:
Ok, will make a PR for that after this is merged

Reply:
I actually think that having the V00 marketplace code is great. It forces us to aways be upgrade ready, and upgradable contract and js code is a part of everything we do and every release. Can't keep things more honest than that.